.azure-pipelines/linux-conda-CI.yml

```diff
@@ -107,6 +111,13 @@ jobs:
       pip install pytest
     displayName: 'install requirements'
 
+  - script: |
```

Comment:
It would be better to take it from pypi because that's a version users will probably use.

Reply:
Yes, that would be preferable. It's not published yet, so we're waiting for that first.

Reply:
Resolved in latest update!